spec/models/integration/post_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'Post show page', type: :feature do
+  let!(:user) { create(:user) }
+  let!(:post) { create(:post, author: user) }
+
+  before do
+    visit post_path(post)
+  end
+
+  it 'displays post details' do
+    expect(page).to have_content("Post id: #{post.id}")
+    expect(page).to have_content("Post title: #{post.title}")
+    expect(page).to have_content("Comments: #{post.comments_counter}")
+    expect(page).to have_content("Likes: #{post.likes_counter}")
+    expect(page).to have_content(post.text)
+  end
+
+  it 'displays "This post has no comments" if there are no comments' do
+    visit post_path(post)
+
+    expect(page).to have_content('This post has no comments')
+  end
+
+  it 'displays a "Like this post" button' do
+    expect(page).to have_button('Like this post')
+  end
+
+  it 'displays a link to create a new comment' do
+    expect(page).to have_link('Create new comment', href: "/comment/new/#{post.id}")
+  end
+end
```

Comment:
- [ ] Great job for implementing some some integration spec for the views. However, according to the project requirements, some tests are yet to be written. Kindly check below requirements and write their respective tests for them. 

### Project Requirements.
![post-index-rqts](https://github.com/theresetuyi/Blog-app/assets/26597432/7d6cdcc7-73f4-40ac-b2b1-aac02ff5583d)